.github/workflows/dev-build.yml

```
@@ -14,14 +14,68 @@ jobs:
     - name: Restore dependencies
       run: dotnet restore ./Apps/Components/Ingrid-Launcher/IngridLauncher.vbproj
+
     - name: Build
       run: msbuild ./Apps/Components/Ingrid-Launcher/IngridLauncher.vbproj -t:rebuild -verbosity:diag -property:Configuration=Release
+    
     - name: Test
       run: dotnet test ./Apps/Components/Ingrid-Launcher/IngridLauncher.vbproj --no-build --verbosity normal
+
+    - name: Determine build result
+      id: result
+      run: |
```

actionlint (1.7.12)

[error] property "build_step" is not defined in object type {read_version: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)